curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646ae2205fe2e9024cc414ed.md

```diff
@@ -0,0 +1,53 @@
+---
+id: 646ae2205fe2e9024cc414ed
+title: Step 7
+challengeType: 0
+dashedName: step-7
+---
+
+# --description--
+
+You will create an HTML `img` tag which will have the `src` attribute set to `an image link here`
```

Comment:
Let's remove this step since this is the fourth project in the curriculum. 

By this point, we are supplying the campers with all of the HTML and CSS.

Reply:
removed